disnake/state.py

```diff
@@ -1218,16 +1220,16 @@ def parse_guild_member_add(self, data) -> None:
             return
 
         member = Member(guild=guild, data=data, state=self)
+
         if self.member_cache_flags.joined:
             guild._add_member(member)
+        self.dispatch("member_join", member)
 
         try:
             guild._member_count += 1
         except AttributeError:
             pass
 
-        self.dispatch("member_join", member)
```

Comment:
What's the reason for moving this?

Reply:
That was an artifact from when I made `raw_member_join`
Arl requested for me to remove it and I guess I just forgot to move that line of code back to where it was

Reply:
53365f9c